libs/backend/src/lib/users/user.controller.ts

```diff
@@ -14,4 +23,11 @@ export class UserController {
   getUser(@Param('userId') userId: string) {
     return this.userService.getuser(userId);
   }
+
+  @Post()
+  // @HttpCode(HttpStatus.CREATED)
```

Comment:
I think this can be uncommented as it wasn't the cause for the pipeline failing. If this causes it to stop working, remove it for now.